examples/decimal-type/decimal_type.bal

```diff
@@ -11,4 +12,8 @@ function floatSurprise() {
 public function main() {
     floatSurprise();
     io:println(nanos);
+
+    // Literals, which belong to type `decimal` use the suffix `d`(`f` suffix is for `float`)
```

Comment:
Could this be misleading to say we always need to use `d`?

Do we need this example here? Given that it includes var also?

Reply:
Having this in the description was not explanatory. That's why added an example. Any suggestion to rephrase this.

Reply:
Shall we at least say 

"can use the `d` suffix" instead of "use the"?